aleemer/Internify
# [IN-125] Create legend on home page to display matches label definition

#### User Story
[[IN-125] Create legend on home page to display matches label definition](https://chaosneutral.atlassian.net/browse/IN-125)

#### Changes made
- Introduced Legend table to the home page so that users will know what the coloured labels mean for the number of matches
- Fixed missing 'Summary' heading in various View page and Create steps
<img width="1437" alt="image" src="https://user-images.githubusercontent.com/37882255/128973370-fadafc50-a21b-4db7-a499-3c09b3e53e8d.png">


#### Does your new code introduce new warnings on dev console?
No

#### Test Steps
1. Login to app
2. On home page, you should see the legend on the top right fixed with vertical scrolling
